src/components/Button/Button.scss

```diff
@@ -8,7 +8,6 @@
   outline: none;
   border: none;
   height: auto;
-  font-family: Roboto, sans-serif;
```

Comment:
because  we use mixins for fonts here for each button size which include font family setting